kitsune/dashboards/tests/test_templates.py

```diff
@@ -35,3 +36,23 @@ def test_render(self):
         eq_(200, response.status_code)
         doc = pq(response.content)
         self._assert_readout_contains(doc, 'unreviewed', unreviewed.document.title)
+
+
+class LocalizationDashVolunteerButtonTests(TestCase):
```

Comment:
I think, you can insert it in ``LocalizationDashTests`` class. having another class seems to be overkill. as its only testing the Dashboard thing